packages/appeals-service-api/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "engines": {
     "node": ">=14.0.0 <15.0.0",
-    "npm": ">=6.0.0 <7.0.0"
+    "npm": ">=6.0.0 <=8.2.0"
```

Comment:
This is out of line with npm in other packages. Why is it changing?

Reply:
This in particular is changing because npm on my machine is past 7.0.0, so had to upgrade in order it to work.